scripts/ansible-2.9.x

```diff
@@ -76,7 +76,8 @@ elif [ -f /etc/debian_version ]; then    # Includes Debian, Ubuntu & Raspbian
 
     echo -e "\napt update; apt install ansible and python3 dependencies\n"
```

Comment:
```suggestion
    echo -e "\napt update; apt install ansible and 7 python3-* dependencies explained at https://github.com/iiab/iiab/tree/master/scripts/ansible.md\n"
```

Reply:
Please also advertise the ansible.md URL within scripts/ansible and scripts/ansible-2.8

Reply:
Also a link from the PRO TIPS section at the bottom of http://FAQ.IIAB.IO Item 11... can't hurt:

"What is Ansible and what version should I use?"
http://wiki.laptop.org/go/IIAB/FAQ#What_is_Ansible_and_what_version_should_I_use.3F

Reply:
I've pushed revised wording.